examples/solids/index.rst

```diff
@@ -0,0 +1,348 @@
+.. _example-petsc-elasticity:
+
+Solid mechanics elasticity mini-app
+========================================
+
+This example is located in the subdirectory :file:`examples/solids`. It solves
+the steady-state static balance momentum equations using unstructured high-order
+finite element/spectral element spatial discretizations. Moreover, the solid
+mechanics elasticity example has been developed using PETSc, so that the
+pointwise physics (defined at quadrature points) is separated from the
+parallelization and meshing concerns.
+
+
+.. _problem-linear-elasticity:
+
+----------------------------------------
+
+The strong form of the static balance linear momentum at small strain for the three-dimensional linear elasticity problem is given by :cite:`hughes2012finite`:
+
+.. math::
+   :label: lin-elas
+   
+   \nabla \cdot \boldsymbol{\sigma} + \boldsymbol{g} = \boldsymbol{0} 
+
+
+where :math:`\boldsymbol{\sigma}`, :math:`\boldsymbol{\epsilon}`
+and :math:`\boldsymbol{g}` are stress, strain, and forcing functions,
```

Comment:
Here it reads "where sigma, epsilon, and g are stress, strain" etc, but strain hasn't been introduced in eq. (24). So the definition of the strain and relative symbol has to be pushed right after eq. (25), where it's introduced for the first time.

Reply:
Or just removed from this sentence, as I see now that it is actually defined on line 50